CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 
 ### Breaking changes
 
+## 1.2.0
```

Comment:
Hey @rubyannmatias, please don't pick a new version. We will do this when a new release will be created, than we are checking all changes and picking the correct new version. For now add this only under the `Unreleased ` section ...